system/lib/standalone/standalone.c

```diff
@@ -116,6 +111,10 @@ __attribute__((__weak__)) int __syscall_fcntl64(int fd, int cmd, ...) {
   return -ENOSYS;
 }
 
+__attribute__((__weak__)) int __syscall_fstat64(int fd, intptr_t buf) {
+  return -ENOSYS;
+}
```

Comment:
Unrelated change?

Reply:
The musl versions call stat on the file before mmap.. so this is now needed to support the musl version.